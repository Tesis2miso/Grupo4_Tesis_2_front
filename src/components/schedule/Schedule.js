import Toolbar from '@mui/material/Toolbar';

function Schedule(props) {
  return (
    <div>
      <Toolbar />
      Schedule
    </div>
  );
}

export default Schedule;
