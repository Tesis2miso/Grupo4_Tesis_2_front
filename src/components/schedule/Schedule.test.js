import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Schedule from './Schedule';

const mockedUsedNavigate = jest.fn();
jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => mockedUsedNavigate,
}));
jest.mock('axios', () => jest.fn());

describe('Tests_Schedules', () => {
  let wrapper = null;

  beforeEach(() => {
    wrapper = shallow(<Schedule loggedIn={true} setToken={() => { }} />);
  });

  test('show schedule', () => {
    expect(wrapper).toMatchSnapshot();
  })
})