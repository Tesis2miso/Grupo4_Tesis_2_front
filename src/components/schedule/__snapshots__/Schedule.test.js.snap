// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests_Schedules show schedule 1`] = `
<div>
  <ForwardRef(Toolbar) />
  Schedule
</div>
`;
